application/common/mail/mfa-recovery.html.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
Since this template is identical to `mfa-manager.html.php` is it necessary? Could we just use one template for both?